app.json

```diff
@@ -0,0 +1,279 @@
+{
+  "name": "spoke",
+  "description": "Spoke",
+  "repository":"https://github.com/moveonorg/spoke",
+  "success_url":"/setup.html",
```

Comment:
can we put envirionment variables in this url?  If so, it might be cool to bring them automatically to the organization creation page (i.e. a first invite) (which maybe makes them a superuser)